src/tools/tests/storage-key-recipe-resolver-test.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Manifest} from '../../runtime/manifest.js';
+import {assert} from '../../platform/chai-node.js';
+import {StorageKeyRecipeResolver} from '../storage-key-recipe-resolver.js';
+
+describe('recipe2plan', () => {
```

Comment:
Update name

Reply:
I have a nested describe block. This way, we can group all r2p tests or just run the tests for this class.

Reply:
Oh wait, I did that in another branch! I'll change this now.